src/test/java/uk/gov/homeoffice/digital/sas/timecard/validators/timeentry/TimeEntryValidatorTest.java

```diff
@@ -50,6 +50,21 @@ void saveTimeEntry() {
                 getAsDate(EXISTING_SHIFT_END_TIME)));
     }
 
+    @Test
+    void validate_startTimeBeforeEndTime_errorReturned() {
```

Comment:
This covers what we expect when there is an endTime and its before the start.
Can we also cover no error cases when there is no end time / there is endTime but its after start. And assert that we **don't** get the "End time must be after start time" message.

Reply:
Would this not be covered by the happy path test cases in the file, where no error is returned for valid cases?

Reply:
The new time entry still has to be valid in those cases too, as in the start time has to be before the end time. I can't think of what test case to add that hasn't been covered?